test/end-to-end/LinkAudit.test.ts

```diff
@@ -93,8 +93,6 @@ test('Changing the link owner should update the link history with Link Owner upd
     .pressKey('ctrl+a delete')
     .typeText(linkTransferField, `${transferEmail}`)
     .click(transferButton)
-  // Close drawer
-  await t.click(closeButtonSnackBar)
```

Comment:
this line is making our e2e tests fail. it's unnecessary, the drawer closes by itself after transferring the link
`closeButtonSnackbar` is actually something else entirely, it refers to this close button:
![Screenshot 2024-01-02 at 11 57 19 AM](https://github.com/opengovsg/GoGovSG/assets/41856541/2f1c07eb-ec42-4d7c-bd8c-0e992192baa2)